.gitignore

```diff
@@ -97,6 +97,8 @@ VMBackup/MANIFEST
 VMBackup/*.pyproj.user
 VMBackup/*.suo
 VMBackup/main/safefreeze/bin/*
+# VMBackup debughelper builds
+VMBackup/debughelper/msft_snap_monit
```

Comment:
is this the output path for binary compilation?

Reply:
Yes. The binary file is by default created in the working directory itself. The name of the binary is defined by the module name which can be found in the first line of `VMBackup/debughelper/go.mod`